plugins/mail/postfix_mail_stats

```diff
@@ -38,7 +38,7 @@ if [ "$1" = "config" ]; then
         echo 'graph_title Postfix Mail Counter'
         echo 'graph_args --base 1000 -l 0'
         echo 'graph_vlabel Hourly Messages'
-        echo 'recieved.label Delivered'
+        echo 'received.label Delivered'
```

Comment:
Updating this will remove history from graphs, as the DS will change name.

Probably worth doing anyway. :)